Doc/library/datetime.rst

```diff
@@ -2020,6 +2020,11 @@ equivalent to ``datetime(*(time.strptime(date_string, format)[0:6]))``, except
 when the format includes sub-second components or timezone offset information,
 which are supported in ``datetime.strptime`` but are discarded by ``time.strptime``.
 
+For :meth:`datetime.strptime` class method without year may fail for Feb 29,
```

Comment:
Please reword this in a shorter, simpler, clearer form. This one long sentence is difficult to understand.

Also, I'm not sure this should be so high this in this list, since it is a rather minor note about an obscure edge case. It should be at the end of this list, or near the end.

Reply:
Moved  near the end of section and reworded to shorter version.